pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyService.java

```diff
@@ -145,8 +143,10 @@ public ProxyService(ProxyConfiguration proxyConfig,
         } else {
             proxyLogLevel = 0;
         }
-        this.acceptorGroup = EventLoopUtil.newEventLoopGroup(1, false, acceptorThreadFactory);
-        this.workerGroup = EventLoopUtil.newEventLoopGroup(numThreads, false, workersThreadFactory);
+        this.acceptorGroup = EventLoopUtil.newEventLoopGroup(proxyConfig.getNumAcceptorThreads(),
+                false, acceptorThreadFactory);
+        this.workerGroup = EventLoopUtil.newEventLoopGroup(proxyConfig.getNumIOThreads(),
```

Comment:
The previous default `numThreads` is `Runtime.getRuntime().availableProcessors()`, but the current default `numThreads` is `Runtime.getRuntime().availableProcessors()`. It has break the default behavior. Do we need to keep the default value?

Reply:
I think we should move to the same default as the broker... primarily because I don't think having a single IO thread is generally something we want to do.